packages/react/.storybook/_container.scss

```diff
@@ -1,9 +1,87 @@
 $css--font-face: true;
+$css--helpers: true;
+$css--body: true;
+$css--use-layer: true;
 $css--reset: true;
+$css--plex: true;
 
 $prefix: 'bx';
 
-// IMPORTANT: This import path should _not_ be used outside our source tree
-// as `src` directory is _not_ meant to be shipped in our NPM package.
-// Use e.g. `@import '~carbon-components/scss/globals/scss/styles.scss'` instead.
-@import '~carbon-components/src/globals/scss/styles.scss'; // SEE THE NOTE ABOVE
+@import '~carbon-components/src/globals/scss/feature-flags';
+@import '~carbon-components/src/globals/scss/vars';
+@import '~carbon-components/src/globals/scss/colors';
+@import '~carbon-components/src/globals/scss/theme';
+@import '~carbon-components/src/globals/scss/mixins';
+@import '~carbon-components/src/globals/scss/layout';
+@import '~carbon-components/src/globals/scss/layer';
+@import '~carbon-components/src/globals/scss/spacing';
+@import '~carbon-components/src/globals/scss/typography';
+@import '~carbon-components/src/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/src/globals/scss/css--reset';
+@import '~carbon-components/src/globals/scss/css--font-face';
+@import '~carbon-components/src/globals/scss/css--helpers';
+@import '~carbon-components/src/globals/scss/css--body';
+@import '~carbon-components/src/globals/grid/grid';
+
+$accordion-type-tokens: map-merge(
+  $tokens,
+  (
+    'body-long-01': $body-long-02,
+  )
+);
+$accordion-heading-height: carbon--mini-units(6);
+@include carbon--type-theme($accordion-type-tokens) {
+  @import '~carbon-components/src/components/accordion/accordion';
```

Comment:
As I mentioned offline, the approach of `@import` of our component Sass file as `@content` conflicts with our `import-once` approach. Probably `@include accordion` instead by making it public?

Reply:
@asudoh that works here. We'd just need to make a pass making sure that every component has a public entrypoint mixin. Most (not all) have entrypoint mixins, and pretty sure they are all marked as private right now. E.g. https://github.com/carbon-design-system/carbon/blob/master/packages/components/docs/sass.md#accordion-mixin

Reply:
BTW I meant:

```scss
@import '~carbon-components/src/components/accordion/accordion';

...

@include carbon--type-theme($accordion-type-tokens) {
  @include accordion;
}
```

(No `@include exports('button')` is needed)

Reply:
@asudoh that would emit the accordion styles twice. I think we're good here with how i have this working with `import-once`. I'll mention this in the recommended approach going forward.

Reply:
@mattrosno Do you mean that the use case is having accordion style _only_ with custom theme?

Reply:
Correct.